src/signals/settings/roles/containers/RoleFormContainer/__test__/RoleFormContainer.test.js

```diff
@@ -120,5 +225,15 @@ describe('signals/settings/roles/containers/RoleFormContainer', () => {
       mapDispatchToProps(dispatch).onSaveRole();
       expect(dispatch).toHaveBeenCalledWith({ type: SAVE_ROLE });
     });
+
+    it('onResetResponse', () => {
+      mapDispatchToProps(dispatch).onResetResponse();
```

Comment:
You should not be testing store functionality in this test; not its responsibility.

Reply:
gedaan